src/components/Menu/index.tsx

```diff
@@ -22,102 +29,125 @@ type MenuProps = {
   setMenuIsOpen?: any;
 };
 
-type MenuState = {
-  isOpen: boolean;
-};
-export default class Menu extends React.Component<MenuProps, MenuState> {
-  constructor(props) {
-    super(props);
-    this.state = { isOpen: true };
-  }
+function Menu(props: MenuProps, ref) {
+  const [isOpen, setIsOpen] = useState(true);
+  const { state } = useLastUpdatedDatesContext();
 
-  componentDidMount() {
-    // We can't do this in the constructor because React will error that the prerendered version is different than live.
-    // Slice off the "@media " string at the start for use in JS instead of CSS
+  useEffect(() => {
     const MQTabletJS = MQTablet.substring(6);
     // If the media query matches, then the user is on desktop and should see the menu by default
-    this.setState({
-      isOpen:
-        typeof window !== 'undefined' && window.matchMedia(MQTabletJS).matches
-    });
-  }
+    setIsOpen(
+      typeof window !== 'undefined' && window.matchMedia(MQTabletJS).matches
+    );
+  }, []);
 
-  closeMenu = () => {
-    this.setState({
-      isOpen: false
-    });
+  useImperativeHandle(ref, () => ({
+    closeMenu: () => closeMenu(),
+    openMenu: () => openMenu()
+  }));
```

Comment:
Since we're using `useForwardRef` on this component, we need to expose the functions that we're trying to share. `SidebarLayout` uses the `closeMenu` and `openMenu` functions